config_defaults/control.ini

```diff
@@ -78,6 +78,10 @@ key_match = docker-autotest
 product = Red Hat Enterprise Linux 7
 component = docker, docker-distribution, docker-registry, atomic,
             rhel-server-atomic, rhel-server-docker, rhel-tools-docker,
+<<<<<<< Updated upstream
```

Comment:
Sorry about that. After I rebased & pushed, I noticed my branch isn't working. Should I remove my commit?